jetty-http/src/main/java/org/eclipse/jetty/http/HttpGenerator.java

```diff
@@ -78,7 +78,7 @@
         FLUSH,                  // The buffers previously generated should be flushed 
         CONTINUE,               // Continue generating the message
         SHUTDOWN_OUT,           // Need EOF to be signaled
-        DONE                    // Message generation complete
+        DONE                    // Generation complete
```

Comment:
This comment difference isn't that clear.

Reply:
The contract for HttpGenerator is that for a given info+content, generate is called repeatedly until DONE is returned, so this is until Generation is complete.   It was wrong to say message generation complete before.  So perhaps "Current phase of generation is complete"?